modules/twinkleprotect.js

```diff
@@ -132,21 +157,42 @@ Twinkle.protect.fetchProtectionLevel = function twinkleprotectFetchProtectionLev
 					expiry: protection.expiry,
 					cascade: protection.cascade === ''
 				};
+				// logs report last admin who made changes to either edit/move/create protection, regardless if they only modified one of them
+				if (!adminEditDeferred) {
```

Comment:
I couldn't quite figure out why this was necessary?  It seems smart, but I couldn't trigger any errors when removing it...